n2v/models/n2v_standard.py

```diff
@@ -209,7 +215,7 @@ def train(self, X, validation_X, epochs=None, steps_per_epoch=None):
         # a masking channel. The N2V_DataWrapper will take care of the pixel masking and manipulating.
         training_data = N2V_DataWrapper(X, np.concatenate((X, np.zeros(X.shape, dtype=X.dtype)), axis=axes.index('C')),
                                                     self.config.train_batch_size, self.config.n2v_perc_pix,
-                                                    self.config.n2v_patch_shape, manipulator)
+                                                    self.config.n2v_patch_shape, manipulator, structN2Vmask=np.array(self.config.structN2Vmask))
```

Comment:
If the `structN2Vmask` is `None` this will be converted to a `array(None, dtype=object)` object. Which is not `None` and therefore the N2V_DataWrapper will try to use the StructN2V code, which will fail in the normal N2V case.